vllm/model_executor/model_loader/loader.py

```diff
@@ -992,6 +993,9 @@ def _load_weights(self, model_config: ModelConfig,
         param_dict = dict(model.named_parameters())
         stacked_quant_state_dict: Dict[str, Dict[int, Any]] = {}
         for quant_param_name in quant_state_dict:
+            if is_pp_missing_parameter(quant_param_name, model):
+                continue
+            
             non_stacked_param_name = quant_param_name
```

Comment:
Thanks for your contibutions. It looks like this is the key of this PR. I added this line on the current main branch, but I couldn't reproduce the ` circular import`

Reply:
I have just realized that PR #9983 has removed the import of `vllm/model_executor/model_loader/loader.py` from `vllm/model_executor/models/utils.py`, so the circular import doesn't happen anymore.

I will modify the request to reflect this. Now perhaps we only need the 3 lines above.

Reply:
I don't get the circular import error anymore using the newest commit, but it still happens in the checks. It seems like the checks are using an older version.

Reply:
Maybe you can using `lazy import`, see:https://github.com/vllm-project/vllm/pull/9765

Reply:
I have updated as you suggested, it's working now, but we should change later to normal import for better clarity as lazy import is not necessary anymore.

Reply:
@mgoin  Could you plz look at this PR?  Thanks~